apis/python/src/tiledbsoma/_query.py

```diff
@@ -526,7 +541,7 @@ def to_anndata(
                     var[name] = var[name].cat.remove_unused_categories()
 
         return AnnData(
-            X=x_future.result(),
+            X=x_future.result() if isinstance(x_future, Future) else x_future,
```

Comment:
Above you load all layers using Dask arrays, but here the test for "Future" is only applied to the unnamed `X`.  What about the `layers` slot below (line 551) - it could be a Future or a Dask array, right?

Reply:
And, will AnnData correctly handle a Dask array in a `layer` slot?